drivers/hmis/config/initializers/hmis_feature.rb

```diff
@@ -23,3 +23,10 @@
   HmisDataCleanup::Util.fix_disabling_condition_nils!
   HmisDataCleanup::Util.fix_race_gender_99s!
 end
+
+TodoOrDie('Remove one-time job', by: '2025-10-06')
+if ENV['ENABLE_HMIS_API'] == 'true'
```

Comment:
```suggestion
if Rails.env.staging? && ENV['ENABLE_HMIS_API'] == 'true'
```

Suggesting to only run this as a TaskQueue in staging, and run it manually in production, more details at https://github.com/open-path/Green-River/issues/7814#issuecomment-3250517530